lib/TextOperations.cpp

```diff
@@ -199,6 +199,44 @@ size_t TextOperations::getUnicodeCharactersCount(const std::string & text)
 	return conv.from_bytes(text).size(); 
 }
 
+std::string TextOperations::shortenUnicodeString(const std::string & input, int maxLength)

```

Comment:
More common name is truncate/truncateUnicodeString

Reply:
Done